wayang-api/pom.xml

```diff
@@ -15,94 +15,109 @@
     <parent>
         <artifactId>wayang</artifactId>
         <groupId>org.apache.wayang</groupId>
-        <version>1.0.0-SNAPSHOT</version>
+        <version>0.6.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
+    <groupId>org.apache.wayang</groupId>
     <artifactId>wayang-api_2.11</artifactId>
+    <version>0.6.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Wayang API Scala-Java</name>
+    <description>wayang implementation of an API of Scala-Java to be enable to work in functional style</description>
+
+    <properties>
+        <java-module-name>org.apache.wayang.api</java-module-name>
+        <wayang.commons.version>0.6.0-SNAPSHOT</wayang.commons.version>
+        <wayang.core.version>0.6.0-SNAPSHOT</wayang.core.version>
+        <wayang.java.version>0.6.0-SNAPSHOT</wayang.java.version>
+        <wayang.spark.version>0.6.0-SNAPSHOT</wayang.spark.version>
+        <spark.version>2.4.0</spark.version>
+        <wayang.sqlite3.version>0.6.0-SNAPSHOT</wayang.sqlite3.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.wayang</groupId>
+                <artifactId>wayang-commons</artifactId>
+                <version>${wayang.commons.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>net.alchim31.maven</groupId>
-                <artifactId>scala-maven-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <artifactId>maven-antrun-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>fake-artifacts</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
-                        <configuration>
-                            <target>
-                                <echo message="Generating fake ${project.build.directory}/${project.artifactId}-${project.version}-sources/javadoc.jar" />
-                                <!--<touch file="${project.build.directory}/target/${project.artifactId}-${project.version}-sources.jar" />-->
-                                <touch file="${project.build.directory}/${project.artifactId}-${project.version}-javadoc.jar" />
-                            </target>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
 
     <dependencies>
         <dependency>
             <groupId>org.apache.wayang</groupId>
             <artifactId>wayang-core</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>${wayang.core.version}</version>
```

Comment:
Using expressions like this "<version>${wayang.core.version}</version>" tend to cause more problems than they solve ... I would strongly reccomend to stick to using plain-text versions. The Release-plugin updates them for you anyway.

Reply:
Ok, I will change it